playwright/register-form-validation.spec.ts

```diff
@@ -7,12 +7,45 @@ interface User {
   password: string;
 }
 
+interface Expectation {
```

Comment:
interfaces dont must be in the files, they must be in types.ts files

Reply:
Should the constants, types and utils from the tests be in the landing? or should I create folders in the e2e tests themselves

Reply:
need to create folders in 'playwright' folder